Dashboard/src/components/Osteppy.js

```diff
@@ -5,45 +5,46 @@ import slack from './assets/slack.svg';
 import ReactMarkdown from 'react-markdown';
 import Moment from 'moment';
 
-const COMPONENT_NAME = "osteppy";
+const COMPONENT_NAME = 'osteppy';
 
 // Reformat Slack messages based on:
 // https://api.slack.com/messaging/composing/formatting#retrieving-messages
 function reformatSlackMentions(text) {
   return text.replace(/<(.*?)>/g, (match, p1) => {
-    switch(p1.slice(0, 2)) {
+    switch (p1.slice(0, 2)) {
       case '@U': // content starting with `@U` or `@W` is a user mention
       case '@W':
         return `**${p1.split('|')[1]}**`;
       case '#C': // content starting with `#C` is a channel link
-        return `**#${p1.split('|')[1]}**`
-      default: // content we don't yet know how to format
+        return `**#${p1.split('|')[1]}**`;
+      default:
+        // content we don't yet know how to format
         console.warn(`Unknown mention: ${match}`);
-        return match
+        return match;
     }
   });
 }
 
 function EODList(props) {
- return (
-   <>
-    <h3>{props.title}</h3>
-    {props.eods.map((eod, i) => (
-      <div key={eod.username + i}>
-        <div className="slack-title">
-          <img className="slack-icon" src={slack} alt="Slack icon" />
-          <span className="github-name">{eod.username}</span>
-          <p className="slack-post"> posted EOD in channel </p>
-          <span className="span-emphasis"> {`#${eod.channel}`}</span>:
+  return (
+    <>
+      <h3>{props.title}</h3>
+      {props.eods.map((eod, i) => (
+        <div key={eod.username + i}>
+          <div className='slack-title'>
+            <img className='slack-icon' src={slack} alt='Slack icon' />
+            <span className='github-name'>{eod.username}</span>
+            <p className='slack-post'> posted EOD in channel </p>
+            <span className='github-repo'> {`#${eod.channel}`}</span>:
```

Comment:
This is probably due to the conflict, but replace `github-repo` with `span-emphasis` since the former class no longer exists in master. Rest looks good.